saltstack/base/salt/ccmv2-inverting-proxy-agent/cdp/bin/ccmv2/run-inverting-proxy-agent.sh

```diff
@@ -6,7 +6,6 @@
 SCRIPT=`basename "$0"`
 BACKEND_ID=$1
 LOG_PATH="/var/log/ccmv2-inverting-proxy-agent.log"
+CONFIG_FILE="/cdp/bin/ccmv2/config-${BACKEND_ID}.toml"
```

Comment:
Now that we know that we would only have a single agent running on a system, can we not suffix with backend id? `config.toml` should be enough for this.